model/src/main/proto/profile.proto

```diff
@@ -44,6 +43,17 @@ message Profile {
   int64 date_created_timestamp_ms = 9;
 }
 
+// Structure for profile avatar image
+message ProfileAvatar {
+  oneof avatar_type {
+    // User selected image or gravatar.
+    string avatar_image_uri = 5;
+
+    // Random color used if user does not provide image
+    int32 avatar_color_hex = 6;
```

Comment:
I think you mean ``avatar_color_rgb`` since this is an integer and not a string ('hex' is really just a way to view an integer, so I'd expect the type of this to be a string if the name indicates hex formatting).